scripts/check_completion_location.py

```diff
@@ -225,6 +225,13 @@ def main(arguments):
     with open(arguments.config_file, 'r') as f:
         field_mappings = json.load(f)
     target_fl = arcgis.features.FeatureLayer(arguments.target_fl, gis)
+    # Check if layer exists
+    try:
+        json = target_fl._lyr_json
```

Comment:
What about `target_fl.properties`? under the hood it should request the json and fail.